dataactcore/config.py

```diff
@@ -108,7 +108,7 @@
     'http://|:(.*)', '', CONFIG_SERVICES['validator_host'])
 # if hosts in config file are set to 0.0.0.0, override to
 # 127.0.0.1 for cross-platform compatibility
-if CONFIG_SERVICES['broker_api_host'] == '0.0.0.0':
+if env is not "local" and CONFIG_SERVICES['broker_api_host'] == '0.0.0.0':
```

Comment:
This check is needed so that config.py will only change the IP of broker to 127.0.0.1 (which doesn't work within docker containers) if it's not running locally and when it's set to '0.0.0.0' .  The main purpose of this is not to change the IP (that is set to 0.0.0.0 in configs) when working locally so that docker can correctly expose ports.